internal/core/data/v2/application/event.go

```diff
@@ -77,21 +90,21 @@ func putEventOnQueue(evt dtos.Event, ctx context.Context, dic *di.Container) {
 		ctx = context.WithValue(ctx, clients.ContentType, clients.ContentTypeJSON)
 	}
 
-	data, err = json.Marshal(evt)
+	data, err = json.Marshal(addEvents)
```

Comment:
@cloudxxx8,  I started look at the App Services side of this change to []dto.AddEventRequest and it isn't pretty. We really need a single DTO. I think this needs more discussion. Wort case for this PR is to just add loop to publish each AddEventRequest  separately.

Reply:
@lenny-intel for normal event (JSON), it's fine with me to publish both single object or array.  However, it will be a problem for CBOR, because it is originally an array and we don't want to encode again here.
We have to resolve the CBOR discussion first.

Reply:
Yes, will bring up in this weeks DS/Core WG meetings.

Reply:
In today's DS WG we decided (tentatively?) to simplify AddEvent to be a single AddEventRequest for both JSON and CBOR. @jpwhitemn do we need to discuss this again in Core WG, or move forward with single event now?

Reply:
@cloudxxx8 - if you agree with reverting to a single Event being handled a single Event request then I think we should proceed without further discussion.   If Cloud believes there are additional consequences, then we should talk about it again in Core.

Reply:
I'm fine with this decision.
However, the change would be large, so I suggest we can merge this PR first.
Jude will open another issue and PR to make the change.

Reply:
@lenny-intel @cloudxxx8 To use a single AddEventRequest DTO for both JSON and CBOR, go-mod-core-contracts needs to be updated, and I've submitted a PR, https://github.com/edgexfoundry/go-mod-core-contracts/pull/430 ,for such change.

Once https://github.com/edgexfoundry/go-mod-core-contracts/pull/430 is merged, I could either update this PR or create a new one for using single AddEventRequest DTO.

Reply:
@lenny-intel @cloudxxx8 I've pushed one more commit https://github.com/edgexfoundry/edgex-go/pull/3002/commits/f2181a9cfd8cd381fbc8312e3f7096b2cceae8a5 for this PR to to use single AddEventRequest.  Swagger file is also updated to reflect the change.